data/2018-10/deploy-stellar-for-norbor.md

```diff
@@ -0,0 +1,34 @@
+---
+id: deploy-stellar-for-norbor-20181003
+date: '2018-10-03'
+time: '18:00 ~ 20:30'
+location:
+  title: BIG Co-working Space
+  url: 'https://www.google.com/maps/place/BIG+Co-working+Space/@13.7566648,100.5697955,17z/data=!3m1!4b1!4m5!3m4!1s0x30e29e8b209f69c3:0xf2124609ad0be030!8m2!3d13.7566648!4d100.5719842'
+categories:
+  - Meetup
+topics:
+  - Stellar
+  - Blockchain
+links:
+  - type: website
+    url: 'https://www.facebook.com/events/797855753879814/'
+    title: Facebook Event
+  - type: ticket
+    url: 'https://www.facebook.com/events/797855753879814/'
+    price: 100 THB
+    detail: จ่ายหน้างาน
+---
+
+# Deploy Stellar for Norbor
+
+> Norbor tam P'Dome tob, ค่าเข้า BIG 100 บาท (ขี้เกียจขอสปอนเซอร์)
+
+ขอคอนเฟิรอมยอดคนมาหน่อนใยครับ ได้จัดที่จัดทางถูก ใครสนใจจริงๆ ให้คง Going ไว้ ใครกะมาไม่สามารถมาได้แน่แล้ว ให้กด Going ออกก่อนได้เลย ขอบคุณครับ
```

Comment:
Can you fix the spellings here? Maybe also link to Facebook event page so that viewers know where to RSVP as “Going”

Reply:
fix typo แล้ว

ส่วนประโยคข้างหลังไม่เข้าใจครับ 555 @dtinth